packages/react-composites/tests/app/chat/index.tsx

```diff
@@ -16,4 +16,9 @@ const params = Object.fromEntries(urlSearchParams.entries());
 initializeFileTypeIcons();
 initializeIconsForUITests();
 
-ReactDOM.render(params.fakeChatAdapterArgs ? <FakeAdapterApp /> : <LiveTestApp />, document.getElementById('root'));
+const domNode = document.getElementById('root');
```

Comment:
Yay, with this code piece, I guess we are good to use newest CRA to create projects!

Reply:
We are!